### DISCUSSION TITLE
What can I do better, as the maintainer for this project?

### DISCUSSION THREAD

**Discussion by snuffyDev** [snuffyDev/Beatbump#204] [Q&A]
Lately I’ve been very busy, personal life and family haven’t been the greatest, and I’ve slacked on Beatbump. Hopefully you guys have noticed the changes to the main instance since im hopefully gearing up to release the refactor soon.

My apologies by the way, for the silence. To be blunt: life has been very difficult lately and after pouring 12+ hours a day into Beatbump and other projects over 2+ years — mentally I’m exhausted as hell. I’m still gratefully working on this project, since I do love it. However it’s been a lot of work and time spent, while also spending around 300-320 the last year to keep it available.

Appreciate yalls support, and thank you for your patience and any feedback you have!

**Comment by davidcollini** [comment]
I've been following your work on Beatbump for a long time and would like to say I appreciate and support the amazing work you've done. I didn't realize you spent so much time on this project per day, if you need a break, don't be afraid to take it, since even though Beatbump is a great project, it's not more important than your mental wellbeing. Going off your last point about how expensive the site is to run, I'd say don't be afraid to ask for donations. There's nowhere on the website that links back to your Github and shows visitors that you're an independent Github developer that is developing this application for no pay and only makes money off of donations. I'd recommend putting a banner explaining your situation and asking for donations at the top of the settings page so it's visible and most users see it, but it's not bothersome. Also open up more ways to donate such as  Monero, Ethereum, Zcash, whatever crypto you want, liberapay, and verify your site on Brave Creators since users on Brave Browser can choose to donate automatically. I hope this paragraph of text was helpful in some way, I just want to see Beatbump be sustainable and I don't want you to worry so much about maintaining this huge project by yourself.

(Also I changed my account, this used to be IronMaltese)

**Comment by davidcollini** [comment]
Also Louis Rossman offers grants to individuals/organizations, this might be an interesting opportunity for the project

https://futo.org/grants/legendary-grants/
